component/azstorage/azstorage.go

```diff
@@ -448,7 +447,7 @@ func (az *AzStorage) ReadInBuffer(options internal.ReadInBufferOptions) (length
 		path = options.Handle.Path
 	} else {
 		size = options.Size
-		path = strings.TrimSpace(options.Path)
+		path = options.Path
```

Comment:
Any reason to not trim spaces here?

Reply:
It is already trimmed earlier, so not doing here